azkaban-web-server/src/test/resources/image_management/image_type_configs.json

```diff
@@ -2,6 +2,6 @@
   "imageType": "configs",
   "description": "configs",
   "deployable": "TAR",
-  "ownerships":[{"owner":  "anath1", "role": "admin"},
-                {"owner":  "apruthi", "role": "admin"}]
+  "ownerships":[{"owner":  "anath1", "role": "ADMIN"},
+                {"owner":  "apruthi", "role": "ADMIN"}]
 }
```

Comment:
Add new line in all the new files in the end.

Reply:
Save action will take care of if automatically. Can you please configure it the way it is mentioned in contributing.md?